src/api/utils/doesConnectorExist.ts

```diff
@@ -0,0 +1,27 @@
+import { Connector } from "@mirohq/websdk-types";
+import { Relation } from "../../models/relation.if";
+
+export default async function doesConnectorExist(
+  startCardId: string,
+  endCardId: string
+) {
+  const connectors = (await miro.board.get({
+    type: "connector",
+  })) as Connector[];
+
+  let connectorExists = false;
+
+  for (const connector of connectors) {
+    const itemData = (await connector.getMetadata("relation")) as Relation;
+
+    if (
+      itemData.startCardId === startCardId &&
+      itemData.endCardId === endCardId
+    ) {
```

Comment:
This raised my suspicion...
It seems that all we get from / store in the `metadata` is what we already have on the connector's default data, namely the ids of the _cards_ which the connector goes to and comes from, which don't need metadata, but are simply `connector.end.item` and `connector.start.item`, aren't they?  

If you're only using the metadata for that, then.. well, you don't need to.  
My original use case would have been to store the ids of the codebeamer _items_ there. But I guess you do the conversion from cb id to card id before, so it would seem you just don't need the connector metadata then.

Reply:
yes you are correct, removed the connector.getMetadata call